engine/server/sv_client.c

```diff
@@ -1289,7 +1289,7 @@ void SV_PutClientInServer( sv_client_t *cl )
 			svgame.dllFuncs.pfnParmsChangeLevel();
 
 			MSG_BeginServerCmd( &msg, svc_restore );
-			Q_snprintf( name, sizeof( name ), "%s%s.HL2", DEFAULT_SAVE_DIRECTORY, sv.name );
+			Q_snprintf( name, sizeof( name ), DEFAULT_SAVE_DIRECTORY "%s.HL2", sv.name );
```

Comment:
Is it really worth it? What if this macro gonna be replaced by a variable.

It's not something that executes every frame so why bother?

Reply:
Ah wait. We probably never gonna change it. 

Nevermind.